ox-typst.el

```diff
@@ -341,7 +341,7 @@ will result in `ox-typst' to apply the colors to the code block."
 
 (defun org-typst-item (item contents info)
   (when-let* ((parent (org-export-get-parent item))
-              (trimmed (org-trim contents)))
+              (trimmed (org-trim (if (arrayp contents) contents ""))))
```

Comment:
```suggestion
              (trimmed (org-trim (if (stringp contents) contents ""))))
```
I would argue that `stringp` is a better choice here. `org-trim` cannot handle vectors (running `(org-trim [77 77])` results in a `(wrong-type-argument stringp [77 77])`).